src/main/java/com/cburch/logisim/circuit/appear/AppearanceSvgReader.java

```diff
@@ -43,31 +43,72 @@
 import com.cburch.logisim.std.io.TtyShape;
 import com.cburch.logisim.std.memory.CounterShape;
 import com.cburch.logisim.std.memory.RegisterShape;
-import java.util.Map;
+import com.cburch.logisim.std.wiring.Pin;
+
+import java.util.List;
 import org.w3c.dom.Element;
 
 public class AppearanceSvgReader {
-  public static AbstractCanvasObject createShape(
-      Element elt, Map<Location, Instance> pins, Circuit circuit) {
-    String name = elt.getTagName();
+  public static class pinInfo {
+    private final Location myLocation;
+    private final Instance myInstance;
+    private Boolean isMapped;
+    
+    public pinInfo(Location loc, Instance inst) {
+      myLocation = loc;
+      myInstance = inst;
+      isMapped = false;
```

Comment:
I see naming inconsequence here: attribute is `isMapped`, setter `setPinIsUsed` and getter `pinIsAlreadyUsed`. Can we have all named in unified way? Say: attribute `isPinMapped`, getter `isPinMapped()` and setter `setPinMapped()`?